packages/owl-bot/src/owl-bot.ts

```diff
@@ -76,15 +78,29 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       if (!installation) {
         throw Error(`no installation token found for ${head.repo.full_name}`);
       }
+      const hasRunLabel = !!context.payload.pull_request.labels.filter(
+        l => l.name === OWLBOT_RUN_LABEL
+      ).length;
```

Comment:
If it's a label event and this label is added, do we want to remove it afterwards (like release-please, kokoro, etc.)

Reply:
that's a good call, didn't think of that.